receiver_custom.html

```diff
@@ -403,7 +406,19 @@ <h3 id= "error_description" class ="cast_text"></h3>
       // prepare the player load script
       var playerJs = document.createElement("script");
       playerJs.type = "text/javascript";
-      playerJs.src = OOYALA_PLAYER_URL + "?" + (debug? ("debug=true&" + v3Version) : v3Version);
+      playerJs.src = OOYALA_PLAYER_URL;
+      if (debug || v3Version) {
+        playerJs.src += "?";
+        if (debug) {
+          playerJs.src += "debug=true";
+          if (v3Version) {
+            playerJs.src += "&" + v3Version;
+          }
+        } else {
+          playerJs.src += v3Version;
+        }
+      }
```

Comment:
This is sample receiver HTML, customers will tune it up for their needs.  Do you believe I should add check whether or not URL constant ends with "?" ?  I think it will be excessive precaution.  Besides, stray question mark at the end of URL does no harm